react/components/Payment/PaymentMethod.tsx

```diff
@@ -67,6 +67,13 @@ const PaymentMethod: FunctionComponent<Props & InjectedIntlProps> = ({
             <InfoIcon colorToken="c-muted-3" />
           </div>
         </div>
+        <div hidden={!isOpen} className="mt2 z-9999 absolute">
```

Comment:
👀 🔫

Reply:
why is this z-index over nine thousand?

Reply:
No particular reason, just to make sure it is front of anything hahaha
I'll bring it down a little